src/main/java/com/google/api/codegen/metacode/FieldStructureParser.java

```diff
@@ -190,12 +181,28 @@ public static InitCodeNode parsePath(InitCodeNode root, Scanner scanner) {
     }
   }
 
-  private static String parseValue(Scanner scanner) {
+  private static String parseKey(Scanner scanner) {
     int token = scanner.scan();
     Preconditions.checkArgument(
         token == Scanner.INT || token == Scanner.IDENT || token == Scanner.STRING,
         "invalid value: %s",
         scanner.input());
     return scanner.tokenStr();
   }
+
+  /**
+   * Parses the RHS of configs. If the RHS is a string literal we return the unquoted string.
```

Comment:
It may just be me, but I find "RHS" somewhat confusing because in the config, this is already inside a yaml RHS. We need to clarify that this is the RHS relative to the `=`, not relative to the yaml `:`.

How about: `Parses the value of the InitCodeNode configuration (ie the RHS of the '=')`